src/openlifu/plan/protocol.py

```diff
@@ -325,11 +325,11 @@ def calc_solution(
             pnp_aggregated = solution.simulation_result['p_min'].max(dim="focal_point_index")
             ppp_aggregated = solution.simulation_result['p_max'].max(dim="focal_point_index")
             # TODO: Ensure this mean is weighted by the number of times each point is focused on, once openlifu supports hitting points different numbers of times
-            intensity_aggregated = solution.simulation_result['ita'].mean(dim="focal_point_index")
+            intensity_aggregated = solution.simulation_result['intensity'].mean(dim="focal_point_index")
```

Comment:
for other SlicerOpenLIFU developers @sadhana-r @arhowe00 note the rename here from 'ita' to 'intensity' -- it will have to be fixed in SlicerOpenLIFU when the `openlifu` version is next bumped